bucket/gitignore.json

```diff
@@ -7,8 +7,8 @@
     "hash": "733e4e2bece6e84ad1d25589cc6d1f852587f2d4a6ffe74b8268435ae229487a",
     "bin": "gitignore.ps1",
     "checkver": {
-        "url": "https://github.com/lukesampson/psutils",
-        "re": "gitignore\\.ps1[\\S\\s]*?/(?<sha>[0-9a-f]{40})[\\S\\s]*?(\\d+)-(\\d+)-(\\d+)",
+        "url": "https://github.com/lukesampson/psutils/commits/master/gitignore.ps1.atom",
+        "regex": "(?s)(\\d+)-(\\d+)-(\\d+).*?(?<sha>[0-9a-f]{40})",
         "replace": "0.${1}.${2}.${3}"
```

Comment:
Just to clarify. Why there is `0.`??

Reply:
Means unreleased version, SemVer's `0.xxx.xxx`.